.swiftlint.yml

```diff
@@ -13,7 +13,7 @@ custom_rules:
     message: "Void return and empty arguments should be avoided in closures."
   closure_opening_space:
     name: Closure Opening Space
-    regex: \{\[|\{\s{2,}\[
```

Comment:
This was incorrectly warning here:
https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/05af2e295710a877395874ecb351bea3c11af3df/Examples/Examples/BasemapGalleryExampleView.swift#L69-L70

Reply:
Thank you for finding and fixing this problem! However, it would be better to make this change in a separate PR.

Reply:
Yep, I've reverted the example changes. I was going to include a SceneView example, but I felt it inflated the scope too much.